backend/routes.js

```diff
@@ -1,17 +1,27 @@
 const express = require('express');
-
 const router = express.Router();
+const User = require('./models/userModel.js');
+const { createUser,login } = require('./controllers/userController.js');
+const bcrypt = require('bcrypt');
+const saltRounds = 12;
+
 
-// sample route
-// a  route like this would fire
-router.get('/', (req, res) => {
-    try {
-        res.json({msg: 'test GET request working'});
-    } catch(error) {
-        res.status(400).json({error: 'something went wrong'});
-    }
+router.post('/newUser', async (req, res) => {
```

Comment:
This isn't the correct endpoint mentioned in the task. The frontend has been set up to have the endpoint "/api/signup" please make this endpoint reflect that

Reply:
fixed, thank you